### PR TITLE
Upgrade to hijackresponse 5

### PR DESCRIPTION
**Pull request by gustavnikolaj**
Using the proposed api from: https://github.com/gustavnikolaj/hijackresponse/pull/24

### CODE REVIEW

**Comment by gustavnikolaj** [comment]
The latest commit updates to the first pre-release of v5. It also removes the err argument to the callback.

File: lib/processImage.js [hunk 1]
```diff
@@ -113,13 +113,9 @@ module.exports = (options) => {
         }
       }
       delete req.headers['if-modified-since']; // Prevent false positive conditional GETs after enabling processimage
-      // hijackResponse will never pass an error here
-      // eslint-disable-next-line handle-callback-err
-      hijackResponse(
+            hijackResponse(
```
Comment (alexjeffburke) [inline lib/processImage.js]: Is there something up with the indentation here?

Comment (gustavnikolaj) [reply lib/processImage.js]: I fixed that in a commit just as you submitted that review :-) https://github.com/papandreou/express-processimage/pull/120/commits/c39ba2b28ab4513cf15e03876ebcabf775549c02

**Comment by papandreou** [comment]
Lookin' good! Also happy to this merge once the api is finalized and 5.0.0 is released!

**Review by papandreou — APPROVED** [review]
LGTM 💯

**Comment by gustavnikolaj** [comment]
This is now ready to be merged and released :-)

**Comment by papandreou** [comment]
Thanks a lot! Good thing I paid for the extended warranty ;)

**Comment by papandreou** [comment]
Released in 10.1.0

**Pull request merged**